src/pilot/deploy-overcloud.py

```diff
@@ -430,19 +815,20 @@ def main():
         start = time.time()
         status = run_deploy_command(cmd)
         end = time.time()
-        print '\nExecution time: {} (hh:mm:ss)'.format(
-            time.strftime('%H:%M:%S', time.gmtime(end - start)))
+        logger.info('\nExecution time: {} (hh:mm:ss)'.format(
+            time.strftime('%H:%M:%S', time.gmtime(end - start))))
         print 'Fetching SSH keys...'
         update_ssh_config()
```

Comment:
This method should only be called after successful deployment. so it should go under "if status == 0:" condition.